lib/rules/selector-max-type/index.js

```diff
@@ -170,7 +175,11 @@ function isDescendantCombinator(node) {
 function isChildCombinator(node) {
   if (!node) return false;
 
-  return isCombinator(node) && node.value.includes(">");
+  return isCombinator(node) && node.value === ">";
```

Comment:
I wonder why this was `.includes` before? 

Do we know if a combinator node can contain whitespace e.g. `/n>`? 

Can you added a test for this please e.g. the following should have no violations:

```
code: ".foo \n+\n a {}"
```

Reply:
Added, it seems to work.